app/src/main/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegration.kt

```diff
@@ -105,6 +112,8 @@ class BrowserToolbarIntegration(
             fragment.showTrackingProtectionPanel()
         }
 
+        setActionButtons(context)
```

Comment:
nit: We could move the "is custom tab" check out of the method to here. `setUrlBackground()` needs it too. So we could check once here and pass it down.